### DISCUSSION TITLE
Add gemini-exp-1206 Model

### DISCUSSION THREAD

**Discussion by ImBinarry** [cline/cline#868] [Feature Requests]
It's an request if you guys can add the new gemini-exp-1206 model, it would be very helpful

**Comment by ryuryuryuryusei** [comment]
agree

**Comment by EriDeLee** [comment]
agree

**Comment by elemental-mind** [comment]
At least Gemini 2.0 experimental landed in the latest update...

**Comment by elemental-mind** [comment]
If you really want this, use Roo Cline

**Comment by saoudrizwan** [comment]
Sorry just now seeing this thread–I should have added this model sooner but I've been a bit out of the loop this last month. I'll be better about adding hardcoded models moving forward.
